### PR TITLE
Coded CV Pull Request

### PR DESCRIPTION
**Pull request by linamoussadek**

### CODE REVIEW

**Review by nalank1 — CHANGES_REQUESTED** [review]
I have left the necessary reviews below

File: lab02/index.html [hunk 1]
```diff
@@ -0,0 +1,354 @@
...
+                        <i class='bx bx-user nav_icon'></i>Profile
+                    </a>
+                </li>
+                <li class="nav_item">
+                    <a href="#education" class="nav_link">
+                        <i class='bx bx-book nav_icon'></i>Projects
+                    </a>
+                </li>
+                <li class="nav_item">
+                    <a href="#skills" class="nav_link">
```
Comment (nalank1) [inline lab02/index.html]: No need for pound key

File: lab02/index.html [hunk 2]
```diff
@@ -0,0 +1,354 @@
...
+                <h2 class="section-title">About</h2>
+                <p class="profile_description">
+                    I was born in Casablanca, Morocco where I attended a French school.
+                    I am looking for a challenging internship for Fall 2024.
+                    I have a Canadian Work Permit and am happy to relocate anywhere!
+                </p>
+            </section>
+
+            <!-- SKILLS  -->
+            <section class="skills section" id="#skills">
```
Comment (nalank1) [inline lab02/index.html:133]: No need for a pound key, you can simply write it as id="skills"

File: lab02/index.html [hunk 3]
```diff
@@ -0,0 +1,354 @@
...
+    <i class="bx bx-up-arrow-alt scrolltop_icon"></i>
+</a>
+
+<!-- HTML2PDF -->
+<script src="assets/js/html2pdf.bundle.min.js"></script>
+
+<!-- MAIN JS -->
+<script src="assets/js/main.js"></script>
+</body>
+<br>
```
Comment (nalank1) [inline lab02/index.html]: There is no need for break, you can discard that

**Review by nalank1 — CHANGES_REQUESTED** [review]
I added my review

File: lab02/index.html (hunk 1 above)
Comment (linamoussadek) [reply lab02/index.html]: Addressed

File: lab02/index.html (hunk 2 above)
Comment (linamoussadek) [reply lab02/index.html:133]: Addressed

**Pull request merged**